lib/standalone.js

```diff
@@ -187,6 +190,22 @@ async function standalone({
 
 	let stylelintResults;
 
+	{
+		/* --ext start */
+		const directories = fileList
+			.filter((p) => fs.existsSync(p) && fs.lstatSync(p).isDirectory())
+			.map((p) => path.relative(globCWD, p));
+		const extList = extensions && extensions.length ? extensions : DEFAULT_EXTENSIONS;
```

Comment:
I need to assign this array before `getConfigForFile` is called so that I use the extensions _after_ [the merger](https://github.com/stylelint/stylelint/pull/6617/files#diff-231823848e62de25699c7b9a8e92b05cc45aaf489c7c1fa417286295d37fcdf6R281).
It will remain a draft until that's fixed.